src/Tools/Console/Command/CommandCompatibility.php

```diff
@@ -9,10 +9,32 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-if ((new ReflectionMethod(Command::class, 'execute'))->hasReturnType()) {
+// Symfony 8
+if ((new ReflectionMethod(Command::class, 'configure'))->hasReturnType()) {
     /** @internal */
     trait CommandCompatibility
     {
+        protected function configure(): void
+        {
+            $this->doConfigure();
```

Comment:
Do we want to add a CI job to get this branch covered?

Reply:
I've changed our existing dev dependencies workflow. It now runs on PHP 8.4 and with Symfony 8 libraries.